packages/anasazi/test/MVOPTester/cxx_main_complex.cpp

```diff
@@ -84,7 +84,7 @@ int main(int argc, char *argv[])
 
 #ifndef HAVE_ANASAZI_TRIUTILS
   cout << "This test requires Triutils. Please configure with --enable-triutils." << endl;
-#ifdef EPETRA_MPI
+#ifdef HAVE_MPI
```

Comment:
Would it be possible to change `HAVE_MPI` to `HAVE_ANASAZI_MPI` throughout Anasazi?

Reply:
I think that's a reasonable request, but that touches a lot of files.  Could I promise to take care of that in a separate pull request?  This one is focused around cleaning up Epetra dependencies and improving Tpetra testing.

Reply:
Sure, a later PR is fine. I'll make sure to remind you ;-)